packages/react-components/review/react-components.api.md

```diff
@@ -249,6 +249,12 @@ export type DefaultMessageRendererType = (props: MessageProps, ids?: {
     messageTimestamp?: string;
 }) => JSX.Element;
 
+// @public
+export interface DominantSpeakers {
+    speakersList: ReadonlyArray<string>;
+    timestamp: Date;
```

Comment:
Is/will `timestamp` used in the components anywhere? If not can we remove it from the react-components API?

Reply:
Not yet, but probably will once we tweak the implementation.

Reply:
Can it be removed until it is needed? UI components don't need to be driven by the same props as state so potentially won't be required (I'm thinking timestamp won't be needed if we want to keep the components using it pure and instead the speakersList would dictate order)

Reply:
Removing it.